eventmesh-test/src/main/java/org/apache/eventmesh/http/demo/sub/controller/SubController.java

```diff
@@ -33,12 +35,17 @@
 
     public static Logger logger = LoggerFactory.getLogger(SubController.class);
 
+    @Autowired
+    private SubService subService;
+
     @RequestMapping(value = "/test", method = RequestMethod.POST)
     public String subTest(@RequestBody String message) {
         logger.info("=======receive message======= {}", JSONObject.toJSONString(message));
         JSONObject result = new JSONObject();
         result.put("retCode", 1);
-        return result.toJSONString();
+        String strResult = result.toJSONString();
+        subService.consumeMessage(strResult);
```

Comment:
The semantics of this line of code looks like it should pass in the received message as a parameter, not the result of consumption

Reply:
Good comment! I will make the change to pass the message instead of result.